apps/studio/components/interfaces/Integrations/Queues/QueuesSettings.tsx

```diff
@@ -0,0 +1,251 @@
+      />
+      <Form_Shadcn_ {...form}>
+        <form id="pgmq-postgrest" onSubmit={form.handleSubmit(onSubmit)}>
+          <FormPanelContainer>
+            <FormPanelContent className="px-8 py-8">
+              <FormField_Shadcn_
+                control={form.control}
+                name="enable"
+                render={({ field }) => (
+                  <FormItem_Shadcn_ className="w-full">
+                    <FormItemLayout
+                      className="w-full"
+                      layout="flex"
+                      label="Expose PGMQ via PostgREST"
+                      description={
+                        <>
+                          <p>
+                            When enabled, you will be able to use any Supabase client library or
+                            PostgREST endpoints to manage your queues
+                          </p>
+                          <p className="mt-1">
+                            The following database functions will be available to use from the{' '}
+                            <code className="text-xs">{QUEUES_SCHEMA}</code> schema:
+                          </p>
+                          <p>
+                            <code className="text-xs">queue_send</code>,{' '}
+                            <code className="text-xs">queue_send_batch</code>,{' '}
+                            <code className="text-xs">queue_read</code>,{' '}
+                            <code className="text-xs">queue_pop</code>,
+                            <code className="text-xs">queue_archive</code>, and
+                            <code className="text-xs">queue_delete</code>
+                          </p>
+                        </>
+                      }
+                    >
+                      <FormControl_Shadcn_>
+                        <Switch
+                          name="enable"
+                          size="large"
+                          disabled={isLoading || !canUpdatePostgrestConfig}
+                          checked={field.value}
+                          onCheckedChange={(value) => field.onChange(value)}
+                        />
+                      </FormControl_Shadcn_>
+                    </FormItemLayout>
+                    {formState.dirtyFields.enable && field.value === true && (
+                      <Admonition type="warning" className="mt-2">
+                        <p>
+                          Queues will be exposed and managed through the{' '}
+                          <code className="text-xs">{QUEUES_SCHEMA}</code> schema
+                        </p>
+                        <p className="text-foreground-light">
+                          Database functions will be created in the{' '}
+                          <code className="text-xs">{QUEUES_SCHEMA}</code> schema upon enabling,
+                          where you can then call through any Supabase client library or PostgREST
```

Comment:
Database functions will be created in the pgmq_public schema upon enabling.  Call these functions via any Supabase client library or PostgREST endpoint to manage your queues. 

?